src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -1147,11 +1146,8 @@ def check_read_nwb(self, nwbfile_path: str):
             data_from_extractor = imaging_extractor.get_video()
             assert_array_equal(two_photon_series.data[:], data_from_extractor.transpose(0, 2, 1))
 
-            assert two_photon_series.description == json.dumps(self.interface.image_metadata)
```

Comment:
This was storing that the full metadata was stored as a description but this is no longer done.